hudi-common/src/main/java/org/apache/hudi/common/table/view/TableFileSystemView.java

```diff
@@ -58,6 +58,11 @@
      */
     Stream<HoodieBaseFile> getLatestBaseFiles();
 
+    /**
```

Comment:
nit: 
```
/**
  * Stream all the latest version data files across partitions with precondition that commitTime(file) before
  * maxCommitTime.
 */
```

More in line with the existing doc. What do you think?

Reply:
make sense!